atomate/vasp/tests/test_drones.py

```diff
@@ -126,5 +128,19 @@ def test_detect_output_file_paths(self):
                               'wavecar': 'WAVECAR.relax1.gz'},
                              doc['calcs_reversed'][1]['output_file_paths'])
 
+    def test_parse_locpot(self):
+        drone = VaspDrone(parse_locpot=True)
+        doc = drone.assimilate(self.Si_static)
+
+        self.assertTrue(drone.parse_locpot)
+        self.assertTrue('locpot' in doc['calcs_reversed'][0]['output'])
+        self.assertTrue(0 in doc['calcs_reversed'][0]['output']['locpot'])
```

Comment:
are these 3 checks needed? seems like the tests afterwards which reference these indices will automatically check for this?

Reply:
(referring to checks of form ``self.assertTrue(0 in doc['calcs_reversed'][0]['output']['locpot'])``

Reply:
You're right that the next three tests cover any issues with the indicies.  These three don't hurt and provide more explicit checks for what is wrong.